### ISSUE TITLE
1.6.2 regression -- file_events table now has OPENED events

### ISSUE THREAD

**Issue by lexelby** [facebook/osquery#1735]
We use the file_events table to do FIM.  It looks like with version 1.6.2, we now get OPENED events logged:

```
Dec 17 16:55:58.866843 hermes.119725@staging.herokudev.com osqueryd[20540]: {"name":"file_events","hostIdentifier":"ip-10-37-166-72","calendarTime":"Thu Dec 17 16:52:23 2015 UTC","unixTime":"1450371143","columns":{"action":"OPENED","atime":"1450368528","category":"binaries","ctime":"1450368528","gid":"0","hashed":"0","inode":"68","md5":"","mode":"0755","mtime":"1450368528","sha1":"","sha256":"","size":"32768","slot":"hermes","target_path":"\/usr\/bin\/","time":"1450369577","transaction_id":"0","uid":"0"},"action":"added"}
```

This results in a torrent of events because we're monitoring /usr/bin and /etc.

Here's our config:

```json
{
  "schedule": {
    "file_events": {
      "query": "select 'hermes' as slot, * from file_events;",
      "interval": 900,
      "removed": false
    },
    "kernel_modules": {
      "query": "select 'hermes' as slot, * from kernel_modules;",
      "interval": 600,
      "platform": "linux",
      "version": "1.4.5",
      "description": "Retrieves all the information for the current kernel modules in the target Linux system.",
      "value": "Only for Linux.  It may pinpoint inserted modules that can carry malicious payloads."
    },
    "kernel_info": {
      "query": "select 'hermes' as slot, * from kernel_info;",
      "interval": 600,
      "version": "1.4.5",
      "description": "Retrieves information from the current kernel in the target system.",
      "value": "Identify out of date kernels or version drift across your infrastructure"
    },
    "os_version": {
      "query": "select 'hermes' as slot, * from os_version;",
      "interval": 600,
      "version": "1.4.5",
      "description": "Retrieves information from the Operative System where osquery is currently running.",
      "value": "Identify out of date operating systems or version drift across your infrastructure"
    }
  },
  "file_paths": {
    "configuration": [
      "/etc/%%"
    ],
    "binaries": [
      "/usr/bin/%%",
      "/usr/sbin/%%",
      "/bin/%%",
      "/sbin/%%",
      "/usr/local/bin/%%",
      "/usr/local/sbin/%%",
      "/opt/bin/%%",
      "/opt/sbin/%%",
      "/opt/%/bin/%%",
      "/opt/%/sbin/%%"
    ],
    "ssh_keys": [
      "/home/%/.ssh/authorized_keys"
    ]
  }
}
```

This regression means that we can't deploy 1.6.2.

**Comment by theopolis** [comment]
Ah! This is because only `IN_ACCESS` is removed from ALL, `IN_OPEN` must also be xored.

**Comment by lexelby** [comment]
confirmed fixed in 1.6.2 :heavy_check_mark:
